robotiq_c_model_control/src/robotiq_c_model_control/robotiq_gripper_action_interface.py

```diff
@@ -27,6 +27,30 @@
               '/0x0F': 'Automatic release completed.',
               }
 
+class RobotiqCommandTimeout(object):
+    """Track command timeouts."""
+    def __init__(self, seconds):
+        """Create a command timer.
+
+        Args:
+            seconds = timeout duration in seconds
+        """
+        super(RobotiqCommandTimeout, self).__init__()
+        self.start_time = rospy.get_rostime()
+        self.duration = rospy.Duration(seconds)
+
+    def start(self):
+        """Start the timer."""
```

Comment:
Although we typically use rospy.Time.now() this version may be more readable as it reports everything in seconds instea d of epoch type

Reply:
So, leave it as it is or change it in to rospy.Time.now()?